Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -0,0 +1,59 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    height: 100%;
+    width: 100%;
+    
+    --color-menu-background: #333333;
+    --color-content-background: #3F3F3F;
+    --color-button-primary: #2C5D87;
+    --color-button-hover: #386082;
+    
+    --font-size-menu: 18px;
+}
+
+.main {
+    flex-direction: row;
+    height: 100%;
+    width: 100%;
+}
+
+.main__menu {
+    width: 240px;
+    padding-top: 20px;
+    background-color: var(--color-menu-background);
+}
+
+.main__content {
+    flex-grow: 1;
+    background-color: var(--color-content-background);
+}
+
+.tab__button {
+    font-size: var(--font-size-menu);
+    -unity-text-align: middle-left;
+    padding: 0 12px 0 20px;
+    margin: 0 0 12px 0;
+    background-color: rgba(0,0,0,0);
+    border-radius: 0;
+    border-width: 0;
+    height: 24px;
+}
+
+.tab__button:hover {
+    background-color: var(--color-button-hover);
+}
+
+.tab__button--selected {
+    background-color: var(--color-button-primary);
+}
+
+.tab__content {
+    display: none;
+}
+
+.tab__content--selected {
+    display: flex;
+}
```

Comment:
`tab__content` is present alongside `tab__content--selected` which implies an ambiguity for which `display` takes precedence. Does USS support combined class references like CSS (`.tab__content.tab__content--selected { ... }`)?

Reply:
it does seem like USS supports it, as a quick demo shows the most specific selector being one higher in the file

![image](https://github.com/aws/amazon-gamelift-plugin-unity/assets/109524503/2a3ff784-bc28-432b-b879-e6a7009be48e)
```css
.tab__button--selected {
    background-color: var(--color-button-primary);
}

.tab__content.tab__content--selected {
    display: flex;
}
```


Are you happy with this:
```css
.tab__content.tab__content--selected {
    display: flex;
}

.tab__button--selected {
    background-color: var(--color-button-primary);
}
```

Reply:
```suggestion
.tab__content {
    display: none;
}

.tab__content.tab__content--selected {
    display: flex;
}
```